frontend/packages/ux-editor/src/containers/DesignView/PageGroupAccordion.tsx

```diff
@@ -79,22 +79,30 @@ export const PageGroupAccordion = ({
         });
       }
     };
-
+    const displayName = group.name || `${t('general.layout_set')} ${groupIndex + 1}`;
+    const selectedGroup = pages.groups.findIndex(
+      (groupp, index) =>
+        selectedItem?.type === 'group' &&
+        (groupp.name || `${t('general.layout_set')} ${index + 1}`) === selectedItem?.id,
+    );
     return (
       <div key={group.order[0].id} className={classes.groupWrapper}>
         <div
-          className={classes.groupHeaderWrapper}
+          className={`${classes.groupHeaderWrapper} ${
+            selectedGroup === groupIndex ? classes.selected : ''
+          }`}
```

Comment:
```suggestion
          className={cn(classes.groupHeaderWrapper, 
          {[classes.selected]: selectedGroup === groupIndex}
          )}
```
It already looks good, so you can decide, but a optional method is to use the javascript utility `import cn from 'classnames'`. https://www.npmjs.com/package/classnames